### ISSUE TITLE
[FEATURE] Process images in subdirectories of input folder

### ISSUE THREAD

**Issue by countbuggula** [ptrsuder/IEU.Winforms#35]
If images are in subfolders, when you run a job it just says "No input images" and finishes processing the queue.
There used to be an output mode "preserve folder structure" but that's not there anymore - was this a feature that was removed?
This would be very helpful in my use case because I have hundreds of images to process but they're in subfolders of ~5-10 each.
Thanks!

**Comment by ptrsuder** [comment]
Preserve folder structure mode should be there, make sure you checked single model.

**Comment by countbuggula** [comment]
Yep, thanks that was the problem.  It'd be nice if you could do a chain AND do subfolders, but I understand if that adds additional complications and this will work for now.  Thanks!

**Comment by ptrsuder** [comment]
Yeah, If I'll return to this, I'll add that
